ci/ci/ci.py

```diff
@@ -207,6 +208,28 @@ async def get_job(request, userdata):
     return await render_template('ci', request, userdata, 'job.html', page_context)
 
 
+@routes.get('/me')
+@web_authenticated_developers_only()
+async def get_user(request, userdata):
+    username = userdata['username']
+    if username in dev_to_gh_user:
+        gh_username = dev_to_gh_user[username]
+    else:
+        return web.Response(text="Please submit a PR to CI with your github username to see your user page!")
```

Comment:
displaying an empty list of dev deploys seems simpler. We should include the GitHub username in the page context, that way folks know if they're not in the mapping or if the mapping is wrong.

Reply:
As in, display a message on the UI above the empty table that they're not in the mapping?

Reply:
I was thinking, the page would just have an empty `<table>`, their username is listed at the top and their `gh_username` is listed, but, perhaps, blank? We could also use `set_message` from `web_common` to signal an error condition to the user.